mindsdb/integrations/handlers/shopify_handler/shopify_tables.py

```diff
@@ -331,3 +333,60 @@ def get_locations(self, **kwargs) -> List[Dict]:
         shopify.ShopifyResource.activate_session(api_session)
         locations = shopify.Location.find(**kwargs)
         return [location.to_dict() for location in locations]
+
+class CustomerReviews(APITable):
+    """The Shopify Customer Reviews Table implementation"""
+
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        """Pulls data from the Yotpo "GET https://api.yotpo.com/v1/apps/{app_key}/reviews?utoken={utoken}" API endpoint.
+
+        Parameters
+        ----------
+        query : ast.Select
+           Given SQL SELECT query
+
+        Returns
+        -------
+        pd.DataFrame
+
+            Shopify customer reviews matching the query
+
+        Raises
+        ------
+        ValueError
+            If the query contains an unsupported condition
+        """
+        select_statement_parser = SELECTQueryParser(
+            query,
+            'customer_reviews',
+            self.get_columns()
+        )
+        selected_columns, where_conditions, order_by_conditions, result_limit = select_statement_parser.parse_query()
+
+        customer_reviews_df = pd.json_normalize(self.get_customer_reviews(limit=result_limit))
+
+        select_statement_executor = SELECTQueryExecutor(
+            customer_reviews_df,
+            selected_columns,
+            where_conditions,
+            order_by_conditions
+        )
+
+        customer_reviews_df = select_statement_executor.execute_query()
+
+        return customer_reviews_df
+
+    def get_columns(self) -> List[Text]:
+        return pd.json_normalize(self.get_customer_reviews(limit=1)).columns.tolist()
+
+    def get_customer_reviews(self, **kwargs) -> List[Dict]:
+        if self.handler.yotpo_app_key is None and self.handler.yotpo_access_token is None:
```

Comment:
Let's use the `or` condition to raise exception also when only one of the keys is provided.
```
if self.handler.yotpo_app_key is None or self.handler.yotpo_access_token is None:
```